packages/backend/src/tokens/handshake.ts

```
@@ -1,4 +1,4 @@
-import { constants, SUPPORTED_BAPI_VERSION } from '../constants';
+import { constants, SUPPORTED_BAPI_VERSION, SUPPORTED_HANDSHAKE_FORMAT, SUPPORTS_HANDSHAKE_NONCE } from '../constants';
```

ESLint

[error] 'SUPPORTS_HANDSHAKE_NONCE' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)